server/src/classes/docking.js

```diff
@@ -20,6 +20,7 @@ export default class DockingPort extends System {
     }
   }
   updateDockingPort({ name, type, clamps, compress, doors, image, docked }) {
+    // Suggestion: change to if(arguments[0].hasOwnProperty('name'))
```

Comment:
Actually, this would not accomplish what I'm intending. It's possible that `name` will be either `null` or `undefined`. `.hasOwnProperty(x)` will return true if `x` is a falsey value, which is not the intended behavior. The only falsey value that should work is `""` (empty string)
```suggestion
```